code/WorkInProgress/Cael_Aislinn/Rust/core_control.dm

```diff
@@ -122,7 +122,7 @@
 	user.set_machine(src)
 
 /obj/machinery/computer/rust_core_control/Topic(href, href_list)
-	..()
+	if(..()) return
```

Comment:
return 1.
Otherwise Topics inheriting from this machine won't get the benefits of the deadcheck.